torch/fx/passes/runtime_assert.py

```diff
@@ -258,14 +258,17 @@ def add_runtime_asserts(ras):
                 # nodes
                 with _set_node_metadata_hook(gm, _node_metadata_hook):
                     res = _sympy_interp(expr_to_proxy, ra.expr).node
+                    expr_msg = f"Runtime assertion failed for expression {ra.expr} on node '{res}'"
+                    assert_msg = (
```

Comment:
we need something like this in inductor  also for torch.compile. maybe put this inside a function in ra to avoid duplication?
see the change in ir.py in this file